beacon_node/beacon_chain/tests/attestation_verification.rs

```diff
@@ -87,7 +87,7 @@ fn get_valid_unaggregated_attestation<T: BeaconChainTypes>(
 
 fn get_valid_aggregated_attestation<T: BeaconChainTypes>(
     chain: &BeaconChain<T>,
-    aggregate: Attestation<T::EthSpec>,
+    mut aggregate: Attestation<T::EthSpec>,
```

Comment:
These were part of the temporary FIXME change, and will trigger warnings now, so should be reverted (I like `git checkout -p` for this)